src/chat/tools/createProject.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  CancellationToken,
+  ChatRequest,
+  ChatResponseStream,
+  LanguageModelTextPart,
+  LanguageModelToolCallPart,
+  LanguageModelToolInvocationOptions,
+  LanguageModelToolResult,
+} from "vscode";
+import { ScaffoldV1Template } from "../../clients/scaffoldingService";
+import { Logger } from "../../logging";
+import { getTemplatesList, scaffoldProjectRequest } from "../../scaffold";
+import { PostResponse } from "../../webview/scaffold-form";
+import { BaseLanguageModelTool, TextOnlyToolResultPart } from "./base";
+
+const logger = new Logger("chat.tools.getTemplateOptions");
+
+export interface ICreateProjectParameters {
+  templateId: string;
+  templateOptions: { [key: string]: string | boolean };
+}
+
+export class CreateProjectTool extends BaseLanguageModelTool<ICreateProjectParameters> {
+  readonly name = "create_project";
+  readonly progressMessage = "Setting up project...";
+
+  async invoke(
+    options: LanguageModelToolInvocationOptions<ICreateProjectParameters>,
+    token: CancellationToken,
+  ): Promise<LanguageModelToolResult> {
+    const params = options.input;
```

Comment:
I'm still not seeing the form auto-populated, and I did some digging here... options seem to only ever include the name. 
<img width="1228" alt="Screenshot 2025-05-12 at 7 09 22 PM" src="https://github.com/user-attachments/assets/17721471-5191-4466-b4dd-135074798542" />

```
getTemplateOptions] PROCESSING TOOL INVOCATION WITH PARAMS: {"templateId":"python-client"}
2025-05-12 19:35:37.501 [debug] [chat.tools.getTemplateOptions] OPTIONS {"input":{"templateId":"python-client"},"toolInvocationToken":{"sessionId":"be53154c-88ba-42d8-bc33-2a53cc9b2442"}}
```

I did find the options listed on the spec which is used in the matching templates logic FYI.

Reply:
`toolInvocationToken` makes sense since that's not part of `input`, but it does look like we need some more/better hinting about passing _some_ kind of values through based on `get_templateOptions`... 🤔